app/src/main/java/kth/jjve/xfran/models/Workout.java

```diff
@@ -1,13 +1,22 @@
 package kth.jjve.xfran.models;
 
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Workout constructor
```

Comment:
I would move the description above the imports. Furthermore, you don't need so many * thingies, but that is just style :)